prac_07/project.py

```diff
@@ -0,0 +1,38 @@
+"""
+prac_07 - project.py
+
+Estimated time: 60 mins
+Actual time: 10 mins
+"""
+
+
+class Project:
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
+        """Initialise Project instance."""
+        self.name = name
+        self.start_date = start_date
```

Comment:
Could we store the `start_date` attribute as a `datetime` object? This might make some other implementations of the program (especially the filtering function,) a bit simpler.

Reply:
You know I was wondering, if there was a better way to sort by the date. Good Idea! 😇